.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: required
+dist: trusty
+language: cpp
+compiler:
+- gcc
```

Comment:
add clang? (should build with 3.7 and 3.8 after #904)
This would check OSX build up to ~95%, without having to setup an OSX travis VM. Btw, can one travis script specify to build on two different VMs (Linux + OSX)?